src/test/java/com/padmaja/stringreplace/EqualityMethodTest.java

```diff
@@ -0,0 +1,38 @@
+package com.padmaja.stringreplace;
+
+import org.junit.Test;
+import java.lang.String;
+
+import static com.padmaja.stringreplace.EqualityMethod.stringCompare;
+import static org.junit.Assert.assertEquals;
+
+//import static com.padmaja.stringreplace.stringReplace.manipulateString;
+//import static org.junit.Assert.*;
+
+ public class EqualityMethodTest {
+
+     @Test
+     public  void  stringCompare1() {
```

Comment:
wrong way to test.
no asserts.
not required system.out  Just for you to look it is good.
to compare many string do it in different test .
And write asserts.

you should have 5 to 6 function to ccompare to test
asserts based on return value.
hat will it return if it is equal, not equal.